### PR TITLE
Add Drone Centre

### PR DESCRIPTION
**Pull request by RealSilverMoon**
See https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14972

### CODE REVIEW

**Comment by boubou19** [comment]
https://github.com/GTNewHorizons/GT5-Unofficial/actions/runs/7044005601/job/19170995159?pr=2386#step:10:2603

File: src/main/java/gregtech/common/GT_Client.java [hunk 1]
```diff
@@ -581,6 +584,7 @@ public void onLoad() {
         super.onLoad();
         new GT_Renderer_Block();
         new GT_MultiTile_Renderer();
+        ClientRegistry.bindTileEntitySpecialRenderer(BaseMetaTileEntity.class, new GT_RenderDrone());
```
Comment (OneEyeMaker) [inline src/main/java/gregtech/common/GT_Client.java:587]: Adding rendering to ***all*** machines just to render one isn't a good choice.

Comment (Connor-Colenso) [reply src/main/java/gregtech/common/GT_Client.java:587]: To get around this issue, you can spawn a block with no hitbox that does this rendering for you. The EOH and space elevator both use this technique.

**Comment by Dream-Master** [comment]
@RealSilverMoon

**Comment by vovaksenov** [comment]
Sorry i havent seen that a pr has already been opened, i posted a comment in the issue a few weeks ago. Basically, I feel like the textures look really out of place. Currently, most textures in the modpack are 16x16 and when you add something else they look strange and make you feel that the pack lacks cohesion. Also the drone color scheme looks almost like a missing texture error, i would use darker colors to avoid this "neon?" effect.

**Comment by RealSilverMoon** [comment]
OK i will try to compress the texture to 16X16. The drone color use large area of white and a bit blue——hopefully to make a sense of "Cyber". Maybe i miss som anti-aliasing on it however.

**Comment by EnderProyects** [comment]
What about using the textures of the drones used in the space elevator, but a little bit modified?

**Comment by vovaksenov** [comment]
I would prefer if the drones used the color palette of most endgame machines like dtpf/eoh/se. Mostly dark blue. If space elevators already have drones (unless you mean the rotating thing on the shaft?), all the more reason to use a similar design.  I dont think the issue is antialiasing or stuff like this, but just that the bright colors (especially the pink part) dont fit.

**Pull request closed without merging**